presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/PruneCountAggregationOverScalar.java

```diff
@@ -55,8 +61,7 @@ public Result apply(AggregationNode parent, Captures captures, Context context)
         for (Map.Entry<Symbol, AggregationNode.Aggregation> entry : assignments.entrySet()) {
             AggregationNode.Aggregation aggregation = entry.getValue();
             requireNonNull(aggregation, "aggregation is null");
-            FunctionCall functionCall = aggregation.getCall();
-            if (!"count".equals(functionCall.getName().getSuffix()) || !functionCall.getArguments().isEmpty()) {
+            if (!"count".equals(functionManager.getFunctionMetadata(aggregation.getFunctionHandle()).getName()) || !aggregation.getArguments().isEmpty()) {
```

Comment:
Any other better way doing this...? cc: @rongrong

Reply:
Maybe add `isCountFunction` to function resolution.

Reply:
I think it probably make sense to add standard aggregation functions such as count to standard function resolution, @rongrong ?